### DISCUSSION TITLE
Desktop app crashing when not in demoMode

### DISCUSSION THREAD

**Discussion by shah-anurag** [spacedriveapp/spacedrive#340] [Community Support] [answered]
When I set demoMode to false([here](https://github.com/spacedriveapp/spacedrive/blob/main/packages/interface/src/App.tsx#L44)), the desktop window closes without any error on Windows machine. Is this an issue faced by others?

**Comment by oscartbeaumont** [comment]
The error will be in the console where you started the app. If you started the app through an exe, it would be a good to start it from a terminal so you can see the error.

**Reply by shah-anurag** [reply]
Hi @oscartbeaumont , thanks for your reply. I am running the app through console but I do not see any error. The desktop app starts and shuts down but the console doesn't exit. Attaching the terminal output below. As you can see, I have manually terminated the process after sometime.

```
Initializing...
Initializing library: be76c56c-85b1-4832-bf6d-61515f3c2e64 C:\Users\Anurag\AppData\Roaming\spacedrive/library.db
"20220526035100_init"
"20220618132721_notes"
"20220624132352_location_node_relation"
Loaded library: ()
Creating node...
node: Data { id: 1, pub_id: "acd41087-48c6-419e-b850-812a7f242f5d", name: "Anurag", platform: 1, version: None, last_seen: 2022-06-29T11:48:52.031+00:00, timezone: None, date_created: 2022-06-29T11:48:52.032+00:00, sync_events: None, jobs: None, location: None }
Spacedrive online
^CTerminate batch job (Y/N)? [0718/233918.273:ERROR:window_impl.cc(114)] Failed to unregister class Cerror: process didn't exit successfully:
```

**Comment by shah-anurag** [comment, marked as answer]
I was gettting this error when i ran the code from terminal
```
Starting node
thread 'main' panicked at 'error while running tauri application: Runtime(CreateWebview(WebView2Error(WindowsError(Error { code: 0x800705B4, message: This operation returned because the timeout period expir }))))', apps\desktop\src-tauri\src\main.rs:133:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `C:\Users\Anurag\Desktop\Personal\Projects\My Projects\spacedrive\spacedrive\target\debug\spacedrive.exe` (exit code: 101)
🚀 A new version of Tauri CLI is available! [1.0.4]
```

I further debugged by opening the dev tool for frontend app(check tauri site on how to do it), I found that the demoMode flag that I was updating was readonly flag, hence it was erroring out.

I updated the flag on object creation and that resolved the issue.
